static/js/libs/colours.js

```diff
@@ -17,6 +17,9 @@ function getColour(holder, imageSelector, imageParentSelector) {
   if (images.length > 0) {
     for (let i = 0, ii = images.length; i < ii; i += 1) {
       const parent = images[i].closest(imageParentSelector);
+      if (!parent) {
+        continue;
```

Comment:
Would be good to avoid arbitrary breaking out of the loop.
This can be easily avoided here by putting the image logic below inside of `if (parent) { ...` instead.